airbyte-integrations/connectors/source-github/source_github/schemas/branches.json

```diff
@@ -3,7 +3,7 @@
   "type": "object",
   "properties": {
     "repository": {
-      "type": ["string"]
+      "type": "string"
```

Comment:
Just wondering, this doesn't functionally change anything as to how it works, right?

Reply:
Also, is `repository` a required field on all of these objects? If so should we declare that in the json schema?

Reply:
2. about `repository`
yes I made it required, except stream Teams stream - `repository` key removed!